.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,7 +15,7 @@ Steps to reproduce the behavior OR commands run:
 3. Enter value '...'
 4. See error
 
-## Expected behaviour
+## Expected behavior
```

Comment:
Set it to US spelling, as I believe most of our texts tends to be. And also it's the spelling used few lines above (the inconistency irked me for a while 😊 ).

Reply:
https://github.com/AbsaOSS/enceladus/pull/192#discussion_r259830693

Reply:
Well, my spellcheck "screams" on _behaviour_, But I don't care if both occurrences will be the same. So it's on @dk1844 to cast the deciding vote, which version it will be 😉

Reply:
😄  I honestly don't care. I just found it funny.

Reply:
My choice generally is the US version, so _behavior_. US-spelling is prevalent in code anyway.